mobile/src/components/common/text/style.ts

```diff
@@ -0,0 +1,101 @@
+import { StyleSheet } from 'react-native';
+import { AppColor } from '~/common/enums/ui/app-color.enum';
+import { AppFont } from '~/common/enums/ui/app-font.enum';
+
+const styles = StyleSheet.create({
+  textGray_400: {
```

Comment:
If you're using this keys for select[Color|Usage] helpers - then it would be better to move this keys to separate enum and use it both for styles obj and inside helpers

Reply:
Same as above